.ruff.toml

```diff
@@ -5,40 +5,30 @@ exclude = [
   "__pycache__",
   "build",
   "sunkit-magex/version.py",
+  "examples/testing/",
 ]
 
 [lint]
 select = ["E", "F", "W", "UP", "PT"]
 extend-ignore = [
   # pycodestyle (E, W)
-  "E501", # LineTooLong                             # TODO! fix
+  "E501", # LineTooLong
```

Comment:
Why have you pulled this comment? Don't we want to fix this everywhere eventually? We have this comment in the template (because we pulled it from core)

Reply:
This comment isn't needed and won't be fixed until we use a formater. Adding a todo here is useless.

Reply:
We have it in core so we should keep it here. This being the whole point of the template.

Reply:
It shouldn't be in core either, nor the template.

Reply:
Then it needs patching in the template and not here.